pandas/tests/dtypes/test_common.py

```diff
@@ -562,6 +562,7 @@ def test_is_bool_dtype():
     assert not com.is_bool_dtype(int)
     assert not com.is_bool_dtype(str)
     assert not com.is_bool_dtype(pd.Series([1, 2]))
+    assert not com.is_bool_dtype(pd.Series(["a", "b"], dtype="category"))
```

Comment:
do we have a categorical case where it is True?

Reply:
We should add something like

``pd.Series([True, False], dtype='category')``